plugins.json

```diff
@@ -128,6 +128,23 @@
             "author": "Ubiquiti Networks, Inc.",
             "zipUrl": "https:\/\/github.com\/Ubiquiti-App\/UCRM-plugins\/raw\/master\/plugins\/revenue-report\/revenue-report.zip"
         },
+        {
+            "name": "routeros",
+            "displayName": "Mikrotik RouterOS",
+            "description": "Plugin for handling RouterOS devices.",
+            "url": "https:\/\/github.com\/Ubiquiti-App\/UCRM-plugins\/tree\/master\/plugins\/routeros",
+            "version": "1.0.0",
+            "ucrmVersionCompliancy": {
+                "min": null,
```

Comment:
`3.0.0` ?

Reply:
nope, it actually can be just deleted, only the `unmsVersionCompliancy` should be present

Reply:
@janprochazkacz ok, so then delete it. "min null and max null" looks like this is compatible with all UCRM versions